service/controller/resource/tccpn/create_test.go

```diff
@@ -0,0 +1,94 @@
+package tccpn
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"testing"
+
+	"github.com/ghodss/yaml"
+	infrastructurev1alpha2 "github.com/giantswarm/apiextensions/pkg/apis/infrastructure/v1alpha2"
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/giantswarm/aws-operator/service/controller/internal/encrypter"
+	"github.com/giantswarm/aws-operator/service/controller/internal/unittest"
+	"github.com/giantswarm/aws-operator/service/controller/resource/tccpn/template"
+)
+
+var update = flag.Bool("update", false, "update .golden CF template file")
+
+var apiWhitelist = APIWhitelist{
```

Comment:
Let's move this to the test case. The struct can be initiatlized there and its value used in the test. No need to "pollute" the package scope.